cars/car-router.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+
+const db = require('../data/dbConfig.js');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    db('cars')
+    .then(listAllCars => {
+      res.json(listAllCars); 
+    })
+    .catch (err => {
+      res.status(500).json({ message: 'Failed to retrieve fruits' });
+    });
+  });
+
+router.post('/', (req, res) => {
+    const carData = req.body;
+    db('cars').insert(carData)
+    .then(ids => {
+      db('cars').where({ id: ids[0] })
+      .then(newCarEntry => {
+        res.status(201).json(newCarEntry);
+      });
+    })
+    .catch (err => {
+      console.log('POST error', err);
+      res.status(500).json({ message: "Failed to store data" });
+    });
+  });
+
+  module.exports = router;
```

Comment:
since you're ahead try and do the other CRUD requests for practice, that's a great post there though, just missing a req.body check to see if the body contains the necessary information to pass to the table